dev/com.ibm.ws.channelfw/src/com/ibm/ws/bytebuffer/internal/PooledWsByteBufferImpl.java

```diff
@@ -44,7 +44,7 @@
     protected int actionState = COPY_ALL_INIT;
     protected final Object actionAccess = new Object();
 
-    private final Object identifier;
+    private final int identifier;
```

Comment:
Are these byte buffer changes related to the shared class cache changes?  Maybe they should go in a separate PR?

Reply:
I update the PR comment to include information that I included some other things found along the way.  Hopefully that satisfies this review comment.

Reply:
Update the PR comment to state that fixing some other things found along the way.